src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,28 @@
-export const TodoFilter = () => (
+type Props = {
+  onChangeQuery: (query: string) => void;
+  onChangeStatus: (status: string) => void;
+  statusOptions: string[];
+  query: string;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  onChangeQuery,
+  onChangeStatus,
+  statusOptions,
+  query,
+}) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
+        <select
+          data-cy="statusSelect"
```

Comment:
You need to manually control value of this select.

Reply:
Done.